src/controllers/auth.controller.js

```diff
@@ -0,0 +1,27 @@
+const authService = require('../services/auth.service');
+
+const signUp = async (req, res) => {
+  try {
+    const { email, password } = req.body;
+
+    await authService.signUp(email, password);
+
+    res.status(201).end();
```

Comment:
statuscode만 반환하는 것도 좋지만, json 형식의 메시지를 클라이언트에게 반환해주세요!

(참고: return 키워드를 사용하면 해당 함수가 return 구문에서 종료가 되고 이후 코드를 실행하지 않습니다.)

```suggestion
    return res.status(201).json({ message: "success" });
```